### DISCUSSION TITLE
Licensing Query

### DISCUSSION THREAD

**Discussion by Minisaints** [Jericho/StrongGrid#480] [General]
Hi,

Is there a copy of the public domain licence available I could get? I might be completely missing it on the README.

Thanks

**Comment by Jericho** [comment]
Here's [my MIT license](https://jericho.mit-license.org/) I used to use back when we had to use our own URL and/or embed a copy of our license in our NuGet packages. But NuGet gained a feature in 2019 that allows package authors to specify the name of the desired license and the NuGet.org web site takes care of displaying a convenient link to the appropriate text. I discontinued using my own custom license during the summer of 2019 and I now rely on NuGet's built-in feature. Here's a screenshot of my package's NuGet page that shows the link I'm referring to:

![image](https://user-images.githubusercontent.com/112710/204823842-58324345-f1b7-4c92-b25e-a3bf09909e2a.png)

**Reply by Minisaints** [reply]
Apologies for the late reply but thank you for this.
